content/en/blog/_posts/2021-05-04-writing-a-controller-for-pod-labels.md

```diff
@@ -0,0 +1,469 @@
+---
+layout: blog
+title: "Writing a Controller for Pod Labels"
+date: 2021-05-04
+slug: writing-a-controller-for-pod-labels
+---
+
+**Authors**: Arthur Busser (Padok)
+
+[Operators][what-is-an-operator] are proving to be an excellent solution to
+running stateful distributed applications in Kubernetes. Open source tools like
+the [Operator SDK][operator-sdk] provide ways to build reliable and maintainable
+operators, making it easier to extend Kubernetes and implement custom
+scheduling.
+
+Kubernetes operators run complex software inside your cluster. The open source
+community has already built [many operators][operatorhub] for distributed
+applications like Prometheus, Elasticsearch, or Argo CD. Even outside of
+open source, operators can help to bring new functionality to your Kubernetes
+cluster.
+
+An operator is a set of [custom resources][custom-resource-definitions] and a
+set of [controllers][controllers] that, when combined, run complex software
```

Comment:
"run complex software" already in prior paragraph.

Anyway, do you really want people to learn that it's complex? The point is that it does something useful, not that it's complicated. I hope :)

Reply:
Fixed 👍